cross-project-dashboard-demo/index.html

```diff
@@ -94,47 +94,65 @@ <h2>{{name}}</h2>
         },
         fetchProjects: function(token) {
           var self = this;
-          $.getJSON(endpoint + 'projects?access_token=' + token)
-            .then(function(results) {
+          $.ajax({
```

Comment:
Nit: since all the network calls pass the same headers, I'd suggest to use a global setup at the start of the file:

```
$.ajaxSetup({
  headers : {
    'X-Rollbar-Access-Token' : token
  }
});
```

That would allow for all the original `$.getJSON` code to remain intact, with a simpler diff.

Reply:
That was my first impression. And I would do that in production code.

However, I believe each example should clearly state all requirements in each case to make it more complete and less confusing.